### PR TITLE
Add bookdown version of this book

### PR DESCRIPTION
**Pull request by xiaosongz**
Source code under /bookdown/ 
Built website under /bookdown/_book/

Currently the r-pkgs bookdown website url still pointing to https://r-pkgs.xiaosongz.com, I will open an issue on the line of url so you can change it once you deployed on your domain.

PDF , Mobi and ePub download option are disabled for now, I will look forward to create other formats later.

Edit link now pointed to `edit: https://github.com/hadley/r-pkgs/edit/master/bookdown/%s`

### CODE REVIEW

**Comment by hadley** [comment]
Can you please replace the existing files rather than creating a new directory? Then I'll tweak the deployment stuff (once I figure out what the new url will be)

**Comment by jennybc** [comment]
Can you remove any `.DS_Store` files from the PR? I see at least one. *(You might want to put `.DS_Store` in your user-level git ignore for future git happiness, but that won't affect this PR.)*

**Comment by xiaosongz** [comment]
> Can you remove any `.DS_Store` files from the PR? I see at least two. _(You might want to put `.DS_Store` in your user-level git ignore for future git happiness, but that won't affect this PR.)_

oops, will address this too.

**Comment by xiaosongz** [comment]
> Can you remove any `.DS_Store` files from the PR? I see at least two. _(You might want to put `.DS_Store` in your user-level git ignore for future git happiness, but that won't affect this PR.)_

I am sorry, I am not very familiar with the **travis CI** configurations and I believe that `.travis.yml` might need to be modified corresponding to new `bookdown` engine?

**Comment by jennybc** [comment]
> I am sorry, I am not very familiar with the travis CI configurations and I believe that .travis.yml might need to be modified corresponding to new bookdown engine?

Oh I think that falls under the parts that @Hadley will look after. We've done a lot of this lately with bookdown and pkgdown. Thanks so much for this transformation!

**Comment by hadley** [comment]
Fixes #464

File: index.rmd [hunk 1]
```diff
@@ -1,27 +1,29 @@
 --- 
 title: "R packages"
-author: "Hadley Wickham"
-date: "`r Sys.Date()`"
+author: ["Hadley Wickham", "Jenny
```
Comment (xiaosongz) [inline index.rmd]: missing paired `]` here ?

Comment (hadley) [reply index.rmd]: Gah, don't know how I managed to delete that

**Comment by hadley** [comment]
Thanks @xiaosongz - this was so helpful!

**Pull request merged**